docs/_integration-with-existing-apps-kotlin.md

```diff
@@ -313,7 +313,11 @@ import android.app.Application;
   @Override
   public void onCreate() {
     super.onCreate();
-+   SoLoader.init(this, OpenSourceMergedSoMapping);
++   try {
++     SoLoader.init(this, OpenSourceMergedSoMapping.INSTANCE);
++   } catch (IOException e) {
++     throw new RuntimeException(e);
++   }
```

Comment:
Why this?

Reply:
Yeah but the IOException catch is not necessary

Reply:
I agree since I catch and just rethrow it, so it is useless but without it the build fails:
error: unreported exception IOException; must be caught or declared to be thrown
      SoLoader.init(this, OpenSourceMergedSoMapping.INSTANCE);

Reply:
I see. That's a bug inside SoLoader and we should fix it rather than having to tweak the documentation

Reply:
See:
- https://github.com/facebook/SoLoader/pull/135